internal/graph/check.go

```diff
@@ -218,21 +220,29 @@ func union(ctx context.Context, concurrencyLimit uint32, handlers ...CheckHandle
 				err = result.err
 				continue
 			}
+			dbReads += result.resp.ResolutionMetadata.DatabaseReads
 
-			if result.resp.GetAllowed() {
+			if result.resp.Allowed {
```

Comment:
Let's add getters to follow suite with the RPC stub design and the other RPC Reqest/Response patterns and usage of getters.